frontend/src/components/userTableRows/userTableRows.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import Checkbox from "../checkbox/checkbox";
+import { Avatar } from "@mui/material";
+import TooltipMessage from "../tooltipMessage/tooltipMessage";
+import { TbDotsVertical } from "react-icons/tb";
+import moment from "moment";
+import "./userTableRows.css";
+import { ROLES, ROLES_ICON } from "../../utils/roles";
+import DropdownMenu from "../dropdownMenu/dropdownMenu";
+import { useOutsideClick } from "@chakra-ui/react";
+
+function UserTableRow({
+  user,
+  checked,
+  changeStatusInTable,
+  updateSelection,
+  rowId,
+}) {
+  const [showMenu, setShowMenu] = useState(false);
+  const [selectUser, setSelectUser] = useState(checked);
+  const ref = React.useRef();
+  useOutsideClick({
```

Comment:
Simple click was not okay ?

Reply:
This was used in order to make dropdown menu disappear  once it is clicked outside of it. If I used onClick the action would happen once I clicked on the component dropdownMenu.